sonosco/modelwrapper.py

```diff
@@ -14,8 +14,13 @@
 
 import torch.distributed as dist
 import torch.utils.data.distributed
-from apex.fp16_utils import FP16_Optimizer
-from apex.parallel import DistributedDataParallel
+
+try:
```

Comment:
Nope, deprecated

Reply:
Please remove any apex functionality from everywhere. No worries, we will reapply it later.

Reply:
okay, we're actually planning to remove the modelwrapper as a whole